### PR TITLE
#120 answer button

### LINKED ISSUE #120: implement the "svara" button on a comment
The respond button needs to visualize and functionality flag the soon to be written response as a response to that comment.

suggestion is to take the now existing input field and add it under the comment with a submit or cancel option

### LINKED ISSUE #189: header in everywhere except bike information has dropped down for some reason

### PR DESCRIPTION
**Pull request by FredSoderberg**
#### What does this PR do?
Adds the option to reply to a comment. Also implements a hide/show button to each comments for their reply
[#120](LINK_TO_ISSUE)

also solves a bug in addbike where location requirement was inverted and the header bug found on certain phones (mine)

##### Why are we doing this? Any context or related work?
One of the last functions missing in the comments section before we can start polishing.

#### Where should a reviewer start?
comments -> bike information
basically check out the new values passed to each comment, including functions and how they are "looped" within the comments.

#### Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
closes #120 
closes #189

### CODE REVIEW

**Comment by jakobemilandersson** [comment]
Having no issues with functionality on my Galaxy A5 or Nexus 5X Emulator! 😄

File: Bikeini/components/Comment.js [hunk 1]
```diff
@@ -115,6 +136,9 @@ export default class Comment extends React.Component {
     this.state = {
       thumbDown: false,
       thumbUp: false,
+      answer: false,
+      replyText: '', // not unsused...just not used here => bikeinformation
```
Comment (jakobemilandersson) [inline Bikeini/components/Comment.js:140]: Eslint giving you a hard time on that one? 😄

File: Bikeini/components/Comment.js [hunk 2]
```diff
@@ -307,10 +331,63 @@ export default class Comment extends React.Component {
...
+        {replyList.length ? (
+          <TouchableOpacity
+            style={styles.separate}
+            onPress={() => this.setState({ showReplys: !showReplys })}
+          >
+            <Text style={styles.answerText}>Show replys</Text>
+          </TouchableOpacity>
+        ) : null}
+      </View>);
+  }
```
Comment (jakobemilandersson) [inline Bikeini/components/Comment.js:355]: Now this is some nice code! 😸

File: Bikeini/navigation/BottomNavigator.js [hunk 3]
```diff
@@ -1,79 +1,82 @@
 import React from 'react';
 import { Platform } from 'react-native';
-import { createStackNavigator, createBottomTabNavigator } from 'react-navigation';
+import { createBottomTabNavigator } from 'react-navigation';
```
Comment (jakobemilandersson) [inline Bikeini/navigation/BottomNavigator.js:3]: Nice! 😮

**Review by jakobemilandersson — APPROVED** [review]
Everything looks nice, good job man! 😸
